presto-tests/src/test/java/com/facebook/presto/execution/TestBeginQuery.java

```diff
@@ -80,12 +81,19 @@ public void setUp()
         getQueryRunner().createCatalog("tpch", "tpch", ImmutableMap.of());
     }
 
-    @BeforeMethod
-    public void beforeMethod()
+    @AfterMethod
```

Comment:
`alwaysRun=true` (highly recommended, even if not strictly necessary in this particular case)